src/test/java/uk/gov/hmcts/reform/sscs/service/AuthorisationServiceTest.java

```diff
@@ -0,0 +1,61 @@
+package uk.gov.hmcts.reform.sscs.service;
+
+import feign.FeignException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.reform.authorisation.ServiceAuthorisationApi;
+import uk.gov.hmcts.reform.sscs.exception.AuthorisationException;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.BDDMockito.then;
+
+@ExtendWith(SpringExtension.class)
+@AutoConfigureMockMvc
+@ActiveProfiles("integration")
+@RunWith(MockitoJUnitRunner.class)
+class AuthorisationServiceTest {
+
+    private static final String AUTHORIZATION = "1203912-39012-=391231";
+
+    @MockBean
+    private ServiceAuthorisationApi serviceAuthorisationApi;
+
+    private AuthorisationService authorisationService;
+
+    @BeforeEach
+    public void setUp() {
+        authorisationService = new AuthorisationService(serviceAuthorisationApi);
+        given(serviceAuthorisationApi.getServiceName(anyString())).willThrow(FeignException.class);

```

Comment:
I think it makes more sense to move the given statements into the respective tests. @BeforeEach is used for general setup between tests. It also makes the test follow the given -> when -> then BDD standard. 
The same applies for the other test